### DISCUSSION TITLE
How do I link a header in a markdown document?

### DISCUSSION THREAD

**Discussion by caiigee** [zed-industries/zed#17417] [Feature Requests]
Hi,

I started writing a markdown text document in Zed and I need to create a link to a section (header).

Here is an example:

```
# Header 1
Lorem ipsum...

# Header 2
Text from [the first header](#header-1) is really interesting!
```

Is this possible in Zed? And if it is what is the correct syntax?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
